.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,91 @@
+name: \U0001F41B Bug report
```

Comment:
These seem to no longer render fine in forms? I guess we can copy the rendered emoji directly like with the ladybug below.

Reply:
```suggestion
name: 🐛 Bug report
```